File: Java NetBeans IDE 8.0/2-Triangulo/src/triangulo/Operaciones.java
package triangulo;

import javax.swing.JOptionPane;

/**
 *
 * @author Benjamin
 */
public class Operaciones {
    private float l1,l2,l3;
    public void Inicilizar(){
        try {
            this.l1=Float.parseFloat(JOptionPane.showInputDialog("Ingresa primer lado: "));
            this.l2=Float.parseFloat(JOptionPane.showInputDialog("Ingresa segundo lado: "));
            this.l3=Float.parseFloat(JOptionPane.showInputDialog("Ingresa tercer lado: "));
        } catch (Exception e) {
            System.out.println("Error al ingresar datos, intenta de nuevo.");
        }
    }
    public void LadoMayor(){
        if(l1>=l2 && l1>=l3){
            System.out.println("El lado mayor es "+l1);
        }
        else if(l2>=l1 && l2>=l3){
            System.out.println("El lado mayor es "+l2);
        }
        else if(l3>=l1 && l3>=l2){
            System.out.println("El lado mayor es "+l3);
        }
        else System.out.println("Lados iguales");
    }
    public void Equilatero(){
        if (l1==l2 && l1==l3) {
            System.out.println("Este tringulo es equilatero");
        }
        else System.out.println("Este no es un triangulo equilatero");
    }
}

File: Python With PyCharm/2-Triangulo/Operaciones.py
l1=0
l2=0
l3=0
def Inicializacion():
    global l1,l2,l3
    l1=float(input("Ingresa el primer lado: "))
    l2=float(input("Ingresa el segundo lado: "))
    l3=float(input("Ingresa el tercer lado: "))
    if not type(l1) is float or not type(l2) is float or not type(l3) is float:
        raise TypeError("Solo se pueden ingresar datos numericos")

def Mayor():
    if l1>=l2 and l1>=l3:
        print("El lado mayor es: ",l1)
    else:
        if l2>=l1 and l2>=l3:
            print("El numero mayor es: ",l2)
        else:
            if l3>=l1 and l3>=l2:
                print("El numero mayor es: ",l3)

def Equilatero():
    if l1==l2 and l1==l3:
        print("El Triangulo es equilatero")
    else:
        print("Este Triangulo no es equilatero")